docs/user-guide/get-started/adequacy-criterion.md

```diff
@@ -47,3 +47,5 @@ Several log files are written:
 
 - `reportbenders.txt` gives information on the progress of the algorithm with an operational perspective,
 - `benders_solver.log` contains more detailed information on all data of interest to follow the progress of the algorithm (`lambda_min`, `lambda_max`, detailed solving times, ...).
+- New output file `criterions.txt` is added under `lp/` dir to store adequacy criterions for all valid patterns (area+criterion)
```

Comment:
In this file remove `New output file`, just say `The file xxx, under lp/ dir stores xxx`, same for the next one as this doc will stay the same in future versions